docs/gui/menus.md

```diff
@@ -272,12 +272,15 @@ A `MenuProvider` is an interface that contains two methods: `#createMenu`, which
 
 A `MenuProvider` can easily be created using `SimpleMenuProvider`, which takes in a method reference to create the server menu and the title of the menu.
 
+##### Open a menu
+
+Use `IForgeServerPlayer#openMenu(MenuProvider, BlockPos)` if you just need to send a pos to create menu.
+Use `IForgeServerPlayer#openMenu(MenuProvider, Consumer<FriendlyByteBuf>)` for more information.(https://github.com/MinecraftForge/MinecraftForge/issues/3228)
+
```

Comment:
This still doesn't explain anything. The position specifies where the menu location is. If the player gets too far away from the menu location, it usually closes. As for the other, describe a use case, do not link to a random issue.